sklearn/utils/estimator_checks.py

```diff
@@ -1323,6 +1327,32 @@ def check_estimators_partial_fit_n_features(name, estimator_orig):
         estimator.partial_fit(X[:, :-1], y)
 
 
+@ignore_warnings(category=(DeprecationWarning, FutureWarning))
+def check_classifier_multioutput(name, estimator_orig):
+
+    estimator_orig = clone(estimator_orig)
+    X, y = make_multilabel_classification(random_state=42)
+    if name not in ['OneVsRestClassifier', 'RidgeClassifierCV']:
+        estimator_orig.fit(X, y.astype(str)).predict(X)
+    estimator_orig.fit(X, y).predict(X)
```

Comment:
Can we check the shape at least to some degree?
Also, I'd really like to avoid hard-coding any estimators in the tests.

Why are we not testing ``decision_function`` and ``predict_proba`` here?

Reply:
I added some checks. Please review.